string name
string family
int64 mac_address
